### ISSUE TITLE
ReplaceToBiomeName and islands do not work in forge

### ISSUE THREAD

**Issue by KPWhiver** [MCTCP/TerrainControl#276]
Hi,

I wrote about this issue in a comment of a different issue, but I was not sure you found it, so here it is again:

When an island biome is made to spawn in a custom biome with ReplaceToBiomeName, the server crashes. The problem only occurs when using forge.

The version of TC used is build #19. The version of forge is 10.12.1.1060 (Minecraft 1.7.2). The issue did not occur when using an older version of TC.

The following crash report is produced:
http://pastebin.com/rTjnzWMX

Here is a configuration with one custom biome and one edited default biome that has the mentioned problems:
WorldConfig.ini
http://pastebin.com/RidauEWf
ForestMod.ini
http://pastebin.com/10Ykwuez

The IsleInBiome line in ForestHills.bc was changed to read:
IsleInBiome: Forest, ForestMod

In short, to reproduce:
- Install forge 10.12.1.1060
- Install TC
- Use the mentioned WorldConfig.ini and ForestMode.ini
- Modify the IsleInBiome line in ForestHills.bc
- Server should now crash

If I've missed some important detail or if you need more information, let me know.

KPW
